source/Patches/CustomOption/Generate.cs

```diff
@@ -75,6 +75,7 @@ public static class Generate
         private static CustomHeaderOption Lovers;
         public static CustomToggleOption BothLoversDie;
         public static CustomNumberOption LovingImpostorOn;
+        public static CustomToggleOption VotedLover;
```

Comment:
Suggestion: This name doesn't actually describe what it does. Consider "CanLoverReportPartner" or "DisallowLoverReportPartner" or something similar?

Reply:
CantReportVotedLover no ?

Reply:
It's only when a lover was voted, the second lover die without body to prevent report

Reply:
Is there any reason to have that off?

Reply:
Have that off what ?